eng/common/pipelines/codeowners-linter.yml

```diff
@@ -0,0 +1,47 @@
+# Lint the CODEOWNERS file for a given repository and filter out baseline errors
+# Note: Due to the nature of the verifications, which includes source paths/globs
+# for the repository, this pipeline cannot use sparse-checkout.
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - .github/CODEOWNERS
+      - .github/CODEOWNERS_baseline_errors.txt
+      - eng/common/pipelines/codeowners-linter.yml
+
+stages:
+- stage: Run
+  variables:
+    skipComponentGovernanceDetection: true
```

Comment:
This is probably OK for now, but we might need to change this to import the /eng/pipelines/templates/variables/globals.yml in each repo as some other repos need to disable other checks in their repos.

Reply:
@weshaggard while I agree, not every repository has or needs a globals.yml. azure-sdk and azure-sdk-for-c are both without. It was easier just to remove the skipComponentGovernanceDetection here, for just this yml file since that's the only thing I can guarantee and control.

Reply:
I suspect you might hit an issue in the .NET repo as we also have to turn off the nuget analysis task. However you could also set that variable in the DevOps UI if that is easier.